plugins/common/starlark/starlark.go

```diff
@@ -133,7 +133,10 @@ func (s *StarlarkCommon) Call(name string) (starlark.Value, error) {
 	if !ok {
 		return nil, fmt.Errorf("params for function %q do not exist", name)
 	}
-	rv, err := starlark.Call(s.thread, fn, args, nil)
+	return starlark.Call(s.thread, fn, args, nil)
+}
+
+func (s *StarlarkCommon) LogError(err error) {
 	if err != nil {
```

Comment:
I think this can be removed as you can assume that there is an error.

Reply:
@srebhan fixed